apps/src/templates/manageStudents/MoveStudents.jsx

```diff
@@ -105,12 +105,7 @@ class MoveStudents extends Component {
     }),
 
     // redux provided
-    sections: PropTypes.arrayOf(
-      PropTypes.shape({
-        name: PropTypes.string.isRequired,
-        id: PropTypes.number.isRequired
-      }).isRequired
-    ),
+    sections: PropTypes.object.isRequired,
```

Comment:
:laughing: Turns out it's much easier to not enforce this?

Reply:
hahah yes, that, and the shape changed from an array to an object and i maybe forgot to update it to something more explicit 🙃

Reply:
This could become
```
PropTypes.objectOf(PropTypes.shape({
  name: PropTypes.string.isRequired,
  id: PropTypes.number.isRequired,
  ...
}));
```
you don't need to name all the fields, only the ones your component depends on.